project/Build.scala

```diff
@@ -79,7 +83,9 @@ object DottyBuild extends Build {
 
       tuning ::: agentOptions ::: travis_build ::: fullpath
     }
-  )
+  ) ++ (if (isTravisBuild) Seq() else addCommandAlias("test", "partest"))
```

Comment:
the main reason is that I use javaagent to attach to JUnit tests from sbt (see `agentOptions`), and attaching to partest doesn't make much sense.